pyproject.toml

```diff
@@ -124,5 +117,29 @@ plugins = ["pydantic.mypy"]
 [tool.pytest.ini_options]
 markers = [
     "integration: marks tests as integration (deselect with '-m \"not integration\"').",
-    "asyncio: marks tests as async."
+    "asyncio: marks tests as async.",
+]
+
+[tool.ruff]
+src = ["truss", "truss-chains", "truss-utils"]
+target-version = "py38"
+line-length = 88
+lint.extend-select = ["I"]
+lint.ignore = ["E402"]
+
+[tool.ruff.lint.isort]
+# Matches the Google Python Style Guide.
+section-order = [
+    "future",
+    "standard-library",
+    "third-party",
+    "first-party",
+    "local-folder",
 ]
+
+[tool.ruff.lint.extend-per-file-ignores]
+"**tests/samples.py" = ["I", "F"]
+"config.py" = ["I", "F"]
```

Comment:
Could you comment what the I and F value mean here.

Reply:
https://github.com/basetenlabs/truss/pull/971/commits/1d744d87b1260206d6dd63dfd85abe5320a5e099